draft-quic-atsss-reqs.mkd

```diff
@@ -306,19 +312,41 @@ This approach provides 0-RTT (Zero Round-Trip Time) conversion service since no
 
 # ATSSS Version 2
 
-The MPTCP-based ATSSS discussed in the previous section is specific to TCP, obviously. In particular, it does not provide a similar feature for UDP flows that would require such service. A UDP flow can be defined as series of UDP packets that have the same source and destination address and port numbers.
-
-ATSSS version 2 aims at providing the same service as ATSSS version 1, i.e.,
-splitting/switching/steering to non-TCP flows. Such flows include
-UDP flows, QUIC connections, DTLS or IPsec connections between the UE and a remote
-host. 
+The MPTCP-based ATSSS discussed in the previous section is specific to TCP,
+obviously. Therefore it does not provide a similar feature to non-TCP traffic,
+in particular not to the increasing share of UDP traffic due to QUIC
+introduction {{QUIC-Deployment}}. A future ATSSS would lose its impact if the
+scope does not become extended beyond support for TCP only.
+
+However, UDP compared to TCP has different dimensions to be considered in
+respect to the ATSSS high-layer approach.
+
+1. UDP consists of a mix of traffic with different demands on transport
+2. UDP is stateless and non-reliable
+
+That does not allow a simple defintion of a multipath transport protocol for UDP
+like it was done for TCP purposes. Even if one would limit to support the QUIC,
+as dominating part of the UDP share, only, this raises issues by its nature of
+an end-to-end encrypted protocol. A potentially upcoming Multipath QUIC
+{{I-D.deconinck-quic-multipath}} cannot be used in the ATSSS architecture like
+the MPTCP in {{fig-mptcp-arch}}, as it dowes not allow interception nor any
+other type of traffic management in an ATSSS UPF.
```

Comment:
This is solution oriented, IMO. It depends how I-D.deconinck is used.

Reply:
OK, maybe that needs clarification. What I meant was that an e2e (UE<->Internet Service) MP-QUIC cannot be intercepted and converted like a e2e MPTCP.

Reply:
Clarified in https://github.com/obonaventure/draft-quic-atsss-reqs/pull/29/commits/c9814766ec725d0dcca6e46ff7452ccd60a6f1de